src/components/navbar.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Rockets',
+    },
+    {
+      id: 2,
+      path: '/Missions',
+      text: 'Missions',
+    },
+    {
+      id: 3,
+      path: 'MyProfile',
+      text: 'MyProfile',
+    },
+  ];
```

Comment:
- [ ] There is no need to implement an array to hold the links, you could accomplish this by just adding the `path /` in the `to=/myprofile` attribute/props of the `<NavLink >` tag :heavy_check_mark:

Reply:
if we do so ! how can we show the names of the pages in the nav bar? @karlavdelgadof

Reply:
what i made is 
im making a object inside that object there is the id and path and name of the nav link 
and them i loop on the object and create the links dynamically using map()

Reply:
> if we do so ! how can we show the names of the pages in the nav bar? @karlavdelgadof

You could just add it like this `<NavLink some attributes=...>My Profile</NavLink>`

Reply:
> what i made is im making a object inside that object there is the id and path and name of the nav link and them i loop on the object and create the links dynamically using map()

Alright @Alzubair98 we can try this way too, no problem :+1: